server/django_server/load_csv_to_db.py

```diff
@@ -41,3 +42,24 @@ def populate_items(csv_file, collection_name):
             # If the error is different to duplicate key entry
             if error.details["writeErrors"][i]["code"] != 11000:
                 print("An error occured")
+
+
+def update_items(csv_file, collection_name):
+    collection = get_collection(collection_name)
+    dict_of_records = clean_data(csv_file)
```

Comment:
I'm raising this as a topic of discussion, as opposed to a suggestion for this PR (as this is outside the scope of the PR).
I think that we should implement the clean_data method as a generator (lazy loading) to avoid keeping all the data in memory all the time. I'm not sure if pandas does this out of the box, but it seems to me that when you convert it to a dictionary, we would be essentially converting the whole thing into a data structure that is definitely stored in memory.

What are your thoughts? This might be a good dev story for next sprint.

Reply:
I see what you mean, I have absolutely no clue, but i can start looking into it and let you know the updates. What do you suggest for this user story, should we keep it like that for now?

Reply:
For this user story, it's fine, this is more of a production issue I can see coming.